design/user.go

```diff
@@ -41,6 +41,22 @@ var _ = a.Resource("user", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 		a.Response(d.Unauthorized, JSONAPIErrors)
 	})
+
+	a.Action("listTokens", func() {
```

Comment:
Should be `/users/:identityID/tokens` instead.
Note, it's User**s** controller. Not User.

Reply:
Ah yes, good point as the user controller is used for user-specific operations as opposed to user management.